styles.css

```diff
@@ -0,0 +1,166 @@
+@import url('https://rsms.me/inter/inter.css');
+
+*,
+::before,
+::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.toolbar {
+  position: relative;
+  width: 375px;
+  height: 6715px;
+  display: flex;
+  flex-direction: column;
+  background-image: url("Background.svg");
+  background-repeat: repeat-x;
+  border-color: brown;
+}
+
+.nav-menu {
+  align-items: center;
+  font-size: 20px;
+  position: absolute;
+  left: 30px;
+  right: 0;
+  top: 44px;
+  bottom: 0;
+  width: 375px;
+  height: 44px;
+  overflow: hidden;
+}
+
+.social-nav {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  position: absolute;
+  width: 200px;
+  height: 50px;
+  order: 3;
+  font-size: 20px;
+  margin-top: 5px;
+}
+
+a {
+  font-family: 'Inter', serif;
+  color: blue;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+a:hover {
+  background: deeppink;
+  border-radius: 50%;
+}
+
+.nav-menu a {
+  color: black;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 375px;
+  height: 726px;
+  left: 0;
+  top: 92px;
+}
+
+.primary-text {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 162px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  top: 64px;
+  left: 24px;
+}
+
+h1 {
+  color: #172b4d;
+  font-size: 48px;
+}
+
+h4 {
+  color: green;
```

Comment:
As asked by the previous reviewer please follow Figma design because it is mandatory.
Here the color is not correct the good color is : #36B37F

Figma : 
<img width="297" alt="image" src="https://user-images.githubusercontent.com/7243132/176913813-f845fb23-3f36-4db6-b88c-28f343309f2d.png">

Your

<img width="360" alt="image" src="https://user-images.githubusercontent.com/7243132/176913885-5e9894d3-9bb4-47e4-be96-ff6cd5ff024c.png">